### PR TITLE
Small 0.10 updates

### PR DESCRIPTION
**Pull request by rkuchan**
Removed the TODO from `storage_engine.md` and added a short note (taken from the [blog post](https://influxdata.com/blog/)) about changes to clustering in 0.10 to `09_vs_010`.

### CODE REVIEW

File: content/influxdb/v0.10/concepts/09_vs_010.md [hunk 1]
```diff
@@ -49,4 +49,12 @@ InfluxData recommends [converting](https://github.com/influxdata/influxdb/blob/m
 
 ## Clustering
 
-TODO: how has clustering changed?
+With InfluxDB 0.10, we now have the concept of a [Meta Node](/influxdb/v0.10/concepts/glossary/#meta-node) and a [Data Node](/influxdb/v0.10/concepts/glossary/#data-node).
+By default, all servers in a cluster will act as both.
+However, it’s now possible to separate them.
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/concepts/09_vs_010.md]: 0.9.x allowed data-only nodes. 0.10.0 now also allows a separate meta-only node as well.

File: content/influxdb/v0.10/concepts/09_vs_010.md [hunk 2]
```diff
@@ -49,4 +49,12 @@ InfluxData recommends [converting](https://github.com/influxdata/influxdb/blob/m
...
+This means in very large clusters or configurations where Data Nodes are under heavy load, Meta Nodes can be separated.
+Another configuration that is useful is with two nodes that act as both Meta and Data nodes, while a third, lesser powered node can act only as a Meta node.
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/concepts/09_vs_010.md]: This sentence is confusing, consider rewording.

**Comment by gunnaraasen** [comment]
Had a few comments, otherwise this looks good. +1

**Comment by rkuchan** [comment]
Thanks, @gunnaraasen. I reworded it - agreed on the confusing wording.

**Comment by gunnaraasen** [comment]
+1

**Pull request merged**